### DISCUSSION TITLE
Backend expense project is not running

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1616] [Q&A] [answered]
Hi Team,
I'm getting below error while I'm running the backend.yml file & have changed the IP's in hosted zone as well but while I'm giving the private IP at the time of command it was running & I'm facing the error at disable default node js.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/a5ed5e2b-80dd-4f37-8245-5b4ae23e1717)
work station public IP: 54.161.49.65
Backend public ip: 3.87.43.89
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/8d9c013a-1815-49c3-afce-237b793328e1)

**Comment by sreedharm07** [comment]
Hi @omswetha 
this error is beacuse of out of memory please change backend instance type to t3 small or t2 small
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/62dfa26e-48d8-4ca9-ba96-84628bb00eb7)

**Reply by omdevops99** [reply]
Okay, What about the below error. While I'm trying with DNS. I'm unable to run.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/a2618fa6-4271-45b6-9941-e09bf61cc077)

**Reply by sreedharm07** [reply]
that error is telling that hosts was not mentioned in playbook 
but in your script i seen hosts are mentioned, am trying to check in your workstation not able to login can you please turn on your instance and ping your ip

**Reply by omdevops99** [reply]
workstation ip : 107.23.234.27

**Reply by sreedharm07** [reply, marked as answer]
Hi @omswetha 
you should give **comma** after the dns name if you not give comma ansible looks for file name, thats why its telling no hosts available 
please give comma in command line 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ecde3252-6496-4bd1-b77e-fde8d6726033)

**Reply by omdevops99** [reply]
Okay, Thanks!!
